### PR TITLE
Add clarification regarding which directories are read-only

### PR DESCRIPTION
**Pull request by barnscott**
The current implementation of the docs suggest that a couple directories, like /etc, are read-only. I wanted to get this clarified in the docs; i tried to be more specific about which are read-only and which are not. If you have any suggestions/corrections for this pull-request, let me know.

### CODE REVIEW

**Comment by tpopela** [comment]
Hi @barnscott and thank you for the change! The change looks good to me, but I will defer to @cgwalters or @dustymabe for a final approval.

File: modules/ROOT/pages/technical-information.adoc [hunk 1]
```diff
@@ -34,10 +34,23 @@ whenever you update the base OS image.
...
-On Silverblue, the root filesystem is immutable. This means that `/`, `/usr` 
-and everything below it is read-only.
+On Silverblue, the root filesystem is immutable. This means that the following 
+system directories and everything below them are read-only:
 
-`/var` is where all of Silverblue's runtime state is stored. Symlinks are used 
+* `/`
+* `/usr`
+
+On the otherhand, the following system directories are writable using the live filesystem:
```
Comment (dustymabe) [inline modules/ROOT/pages/technical-information.adoc:43]:
```suggestion
On the otherhand, the following system directories are writable:
```

File: modules/ROOT/pages/technical-information.adoc [hunk 2]
```diff
@@ -34,10 +34,23 @@ whenever you update the base OS image.
...
+
+* `/boot`
+* `/dev`
+* `/etc`
+* `/run`
+* `/sysroot`
+* `/tmp`
+* `/var`
```
Comment (dustymabe) [inline modules/ROOT/pages/technical-information.adoc:51]: There's an effort to make `/boot` and `/sysroot` read-only too (not sure if it made it to silverblue yet). I wouldn't mention `/dev/`. It's not really a filesystem that you would store files in. Might be worth mentioning that `/run/` and `/tmp/` are tmpfs (i.e. just files in RAM).

File: modules/ROOT/pages/technical-information.adoc [hunk 3]
```diff
@@ -34,10 +34,23 @@ whenever you update the base OS image.
...
+system directories and everything below them are read-only:
 
-`/var` is where all of Silverblue's runtime state is stored. Symlinks are used 
+* `/`
+* `/usr`
+
+On the otherhand, the following system directories are writable using the live filesystem:
+
+* `/boot`
+* `/dev`
```
Comment (barnscott) [inline modules/ROOT/pages/technical-information.adoc:46]:
```suggestion
```

File: modules/ROOT/pages/technical-information.adoc [hunk 4]
```diff
@@ -34,10 +34,23 @@ whenever you update the base OS image.
...
+* `/etc`
+* `/run`
+* `/sysroot`
+* `/tmp`
```
Comment (barnscott) [inline modules/ROOT/pages/technical-information.adoc:50]:
```suggestion
```

File: modules/ROOT/pages/technical-information.adoc [hunk 5]
```diff
@@ -34,10 +34,23 @@ whenever you update the base OS image.
...
-`/var` is where all of Silverblue's runtime state is stored. Symlinks are used 
+* `/`
+* `/usr`
+
+On the otherhand, the following system directories are writable using the live filesystem:
+
+* `/boot`
+* `/dev`
+* `/etc`
+* `/run`
```
Comment (barnscott) [inline modules/ROOT/pages/technical-information.adoc:48]:
```suggestion
```

File: modules/ROOT/pages/technical-information.adoc [hunk 6]
```diff
@@ -34,10 +34,23 @@ whenever you update the base OS image.
...
+* `/sysroot`
+* `/tmp`
+* `/var`
+
```
Comment (barnscott) [inline modules/ROOT/pages/technical-information.adoc:52]:
```suggestion
The following system directories are writable, but expect to become read-only in future versions of Silverblue:

* `/boot`
* `/sysroot`

```

File: modules/ROOT/pages/technical-information.adoc [hunk 7]
```diff
@@ -34,10 +34,23 @@ whenever you update the base OS image.
...
+On Silverblue, the root filesystem is immutable. This means that the following 
+system directories and everything below them are read-only:
 
-`/var` is where all of Silverblue's runtime state is stored. Symlinks are used 
+* `/`
+* `/usr`
+
+On the otherhand, the following system directories are writable using the live filesystem:
+
+* `/boot`
```
Comment (barnscott) [inline modules/ROOT/pages/technical-information.adoc:45]:
```suggestion
```

File: modules/ROOT/pages/technical-information.adoc [hunk 8]
```diff
@@ -34,10 +34,23 @@ whenever you update the base OS image.
...
+* `/dev`
+* `/etc`
+* `/run`
+* `/sysroot`
```
Comment (barnscott) [inline modules/ROOT/pages/technical-information.adoc:49]:
```suggestion
```

File: modules/ROOT/pages/technical-information.adoc (hunk 2 above)
Comment (barnscott) [reply modules/ROOT/pages/technical-information.adoc:51]: I incorporated the suggestions.

**Comment by travier** [comment]
Thanks for the PR. I'm updating it in https://github.com/fedora-silverblue/silverblue-docs/pull/89

**Pull request closed without merging**